WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -19,16 +17,31 @@ class AccountSettingsService {
         static let pollingInterval = 60.0
     }
 
+    enum Notifications {
```

Comment:
I can't find the original article but from https://github.com/raywenderlich/swift-style-guide#constants:

> The advantage of using a case-less enumeration is that it can't accidentally be instantiated and works as a pure namespace

Using both `struct` and `enum` purely for the namespacing is a bit of a hack. With an `enum` you can at least avoid misusing the type, since you can't do `AccountSettingsService.Notifications()`